dashboard-client/src/routes/config/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { FunctionalComponent, h } from "preact";
+import { useCallback, useEffect, useState } from "preact/hooks";
+import * as style from "./style.css";
+
+interface Props { }
+
+const Config: FunctionalComponent<Props> = (props: Props) => {
+    const [configString, setConfigString] = useState<string>('');
+
+    useEffect(function componentDidMount() {
+        fetch('http://localhost:3000/config', { method: 'GET' })
```

Comment:
I moved this to [`/iotsec/config`](https://github.com/timothy-b/IOT-SEC/blob/b7e13cf4da2a595343222f5267148275fa13d8ff/src/utilities/server.ts#L50) and gated it behind authentication.

What do you think about having another config file for this app, and moving this baseUrl and the authentication creds there?